Product/Production/Services/DocumentRetrieveCore/src/main/java/gov/hhs/fha/nhinc/docretrieve/entity/OutboundDocRetrieveStrategyBase.java

```diff
@@ -101,26 +102,26 @@ public void execute(OutboundDocRetrieveOrchestratable message) {
     protected abstract RetrieveDocumentSetResponseType callProxy(OutboundDocRetrieveOrchestratable message);
 
     protected void auditRequestMessage(RetrieveDocumentSetRequestType request, String direction,
-            String connectInterface, AssertionType assertion, String requestCommunityID) {
+        String connectInterface, AssertionType assertion, String requestCommunityID) {
         gov.hhs.fha.nhinc.common.auditlog.DocRetrieveMessageType message = new gov.hhs.fha.nhinc.common.auditlog.DocRetrieveMessageType();
```

Comment:
I know this is existing code, but with all the changes that have been made, does this declaration still need to be fully-qualified?